core/src/main/scala/com/commercetools/queue/QueueConfiguration.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2024 Commercetools GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.commercetools.queue
+
+import scala.concurrent.duration.FiniteDuration
+
+final case class QueueConfiguration(messageTTL: FiniteDuration, lockTTL: FiniteDuration)
```

Comment:
It could be worth it to explain the diff between the two options and wether they're supported for each provider or not. In this regard, we may need to have a section with all the gotchas.

Reply:
I will extend this with the addition of the dead letters, I will document all of it then.